index.js

```diff
@@ -19,72 +19,78 @@ var Client = redefine.Class({
 
   request: function(path, options) {
     if (!options) options = {};
-    if (!options.headers) options.headers = {};
```

Comment:
It seems to me that the user cannot provide custom headers anymore. While I cannot see the usecase for that, we are still breaking BC. Is that on purpose?